Zotero/Scenes/AppCoordinator.swift

```diff
@@ -463,24 +463,23 @@ extension AppCoordinator: DebugLoggingCoordinator {
             }
 
             func share(debugId: String, customMessage: String?, userId: Int) {
-                var actions = [
+                let actions = [
                     UIAlertAction(title: L10n.ok, style: .cancel, handler: nil),
-                    UIAlertAction(title: L10n.copy, style: .default, handler: { _ in
+                    UIAlertAction(title: L10n.Settings.CrashAlert.submitForum, style: .default, handler: { _ in
                         UIPasteboard.general.string = debugId
```

Comment:
The `debugId` will be copied to paste board only if the user tries to submit to the forum. It might be better to be copied every time, i.e. just before showing the alert, regardless of the action selected.

Reply:
Makes sense, changed in d219e47b67c93d92933ce16a90215d5bb83168c3